### DISCUSSION TITLE
Custom Preset plugin

### DISCUSSION THREAD

**Discussion by pallardino** [easepick/easepick#61] [Q&A] [answered]
An other question for customising preset-plugin. 

- First, I have tried to customise labels to put it in French. According to the documentation I passed a list of string, but unfortunately I got the following error from my js console : 
`TypeError: this.options.customLabels.forEach is not a function. (In 'this.options.customLabels.forEach(((t,e)=>{this.options.customPreset[t]=s[e]}))', 'this.options.customLabels.forEach' is undefined)`
- Second, I don't really understand how I can define your own ranges. According to the documentation I have to pass an object, but which object ? For exemple, I would like delete all predefined range and create a custom range for a year. 

<img width="446" alt="Capture d’écran 2022-05-13 à 10 21 44 PM" src="https://user-images.githubusercontent.com/39769580/168385220-473deee9-6d0c-41dd-ba83-1bb29cd27d1a.png">

Thanks in advance

**Comment by wakirin** [comment, marked as answer]
I see that this is a bug.
I will fix it later.
Currently you can do something like this: https://jsfiddle.net/waki/zarwbc3d/

**Reply by pallardino** [reply]
Thank you so much

**Reply by avianion** [reply]
@wakirin So for example, how can I change it to display "Aujord Hui" instead of the 2022-08-12 - 2022-08-14 date?

I want the datepicker to simply display "Aujord Hui" in the display section - is there an option for this?
